.github/workflows/security-scans.yml

```diff
@@ -25,13 +25,3 @@ jobs:
 
     - name: Gitleaks - detecting hardcoded secrets
       uses: zricethezav/gitleaks-action@v1.1.4
-
-    - name: Notify slack on failure
-      if: failure()
-      env:
-        SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-      uses: voxmedia/github-action-slack-notify-build@v1
-      with:
-        channel_id: ${{ secrets.SLACK_ALERTS_CHANNEL_ID }}
-        status: FAILED
-        color: danger
```

Comment:
should we also make it a required check on branches? 
<img width="255" alt="Screenshot 2020-10-16 at 10 55 33" src="https://user-images.githubusercontent.com/1931615/96237930-28482c00-0f9e-11eb-95f9-d1987c8a3f5a.png">

Reply:
Yes, but not for now, we'll block all PRs that have to merge the master branch to a PR